tests/test-basic-c.c

```diff
@@ -171,6 +172,30 @@ test_raw_file_to_archive_z2_stream (gconstpointer data)
   g_assert_cmpint (checks, >, 0);
 }
 
+static void
+test_repo_is_created (gconstpointer data)
+{
+  OstreeRepo *repo = OSTREE_REPO (data);
+  gboolean exists = 0;
+  g_autoptr(GError) error = NULL;
+  g_autoptr(GFile) path;
+  int success
+
+  /* Test a return value of TRUE on a repo that exists */
+  success = ostree_repo_is_created (repo, &exists, error);
```

Comment:
I think you want to pass `&error` here. That's what's making the compiler cry :)